src/__functional__/monorepo/getPackageInfos.test.ts

```diff
@@ -1,13 +1,15 @@
 import fs from 'fs-extra';
+import _ from 'lodash';
+import { gitFailFast } from 'workspace-tools';
 import { RepositoryFactory } from '../../__fixtures__/repositoryFactory';
 import { tmpdir } from '../../__fixtures__/tmpdir';
 import { getPackageInfos } from '../../monorepo/getPackageInfos';
 import { PackageInfos } from '../../types/PackageInfo';
 import { getDefaultOptions } from '../../options/getDefaultOptions';
 import { getCliOptions } from '../../options/getCliOptions';
-import { gitFailFast } from 'workspace-tools';
 
 const defaultOptions = getDefaultOptions();
+const repoOptions = require('../../../beachball.config');
```

Comment:
I got rid of this by making `getPackageOptions` ignore CLI and repo options if `process.env.NODE_ENV === 'test'`. That works at the moment because we don't have test coverage for any scenarios where the packages would need to inherit options from the fixture...